### DISCUSSION TITLE
Google translate does not work with this package

### DISCUSSION THREAD

**Discussion by jayleaton** [amannn/next-intl#1189] [General]
I have setup a basic non routing instance. Everything works if I use the accept-language header. We do not need to translate most of out site as this would take litterally months so we rely on google translate and it's mostly good enough. But in some cases we want to replace the values with our own values I was hoping this package would work. It seems to not have any impact. 

We use "sv" as the default language so I have sv.json and en.json files with the correct translations. but the en.json fine has no impact when a user selects to translate with google translate. 

Not sure if this is intended or not but it's quite annoying if this is intended that would mean we need to translate the entire site rather then just manually set the translations for the bad ones ourselves.

**Comment by amannn** [comment]
@oldbettie Is your expectation that when a user uses Google Translate in the browser, that other messages are fetched in response based on the language that the user translates to? That's not quite how it works.

Rather, you should set up a complete `en.json` that contains all the labels that you've picked. Note that you don't have to translate all of this yourself, but you could use a platform like [Crowdin](https://crowdin.com/ai-localization) or the VSCode plugin [i18n Ally](https://github.com/lokalise/i18n-ally/wiki/Machine-Translation) to translate your labels. The major benefit is that you can review the translated labels and adapt them on a case-by-case base. Especially Crowdin supports this use case well by giving non-developers access to translations.

Hope this helps!

**Reply by jayleaton** [reply]
I have created both `sv.json` and `en.json` files but when I translate with google translate from swedish to english the values in the en.json are not used. We never implemented it because it felt like a burden I guess I hoped it worked with the built in browser translator. The issue then becomes an all or nothing issue. if I mark it as english then it assumes its english and it will no longer translate any of the other language into english or even give the tranlate dialog because my browser is default to english and the site says english it assumes its english.

I cant even find a way to view the translation event to trigger the swap to the en.json form

**Reply by jayleaton** [reply]
I would have assumed that all files are sent to the client and when the language is changed then it uses those files.
